### PR TITLE
week1-3-homework-cynthia

### PR DESCRIPTION
**Pull request by cyeburu**

### CODE REVIEW

**Comment by cyeburu** [comment]
week 3 updated

File: week-3/Homework/mandatory/7-password-validator.js [hunk 1]
```diff
@@ -36,22 +36,53 @@ PasswordValidationResult=  [false, false, false, false, true]
...
+function checkPasswords(arrPassword) {
+  let checkLowercase = /[a-z]/
+  let checkUppercase = /[A-Z]/
+  let checkNumber = /[0-9]/
+  let checkSymbol = /[!#$%.]/
+  if (checkLowercase.test(arrPassword) &&
+    checkUppercase.test(arrPassword) &&
+    checkNumber.test(arrPassword) &&
+    checkSymbol.test(arrPassword) &&
+    arrPassword.length >= 5
```
Comment (sabia-m) [inline week-3/Homework/mandatory/7-password-validator.js]: the layout of this is really nice!

File: week-3/Homework/mandatory/3-space-colonies.js [hunk 2]
```diff
@@ -2,16 +2,25 @@
...
-
+// function colonisers(oldArr) {
+//   let initialArr1 = oldArr.filter(e => e < "B")
+//   let finalArr2 = initialArr1.filter(e => e !== "Aldous" && e !== "Asimov")
+//   //console.log(newArr)
+//   return finalArr2
+// }
+function colonisers(oldArr) {
+  let initialArr1 = oldArr.filter(e => e.charAt(0) === "A" && e.includes("family"))
+  return initialArr1.map(e => e)
```
[the remark this answers is not in the document]
Comment (cyeburu) [reply week-3/Homework/mandatory/3-space-colonies.js]: Hi Frosas,
Thank you for taking your time to review my code. I have action the point highlighted

[the remark this answers is not in the document]
Comment (frosas) [reply week-3/Homework/mandatory/3-space-colonies.js]: Excellent. Thanks for updating the code 👍

**Pull request closed without merging**